region/lowernorfair/east.json

```diff
@@ -3027,6 +3043,270 @@
+                      ]}
+                    ]
+                  }},
+                  {"obstacle":{
+                    "id": "A",
+                    "requires": [
+                      {"or":[
+                        "canUsePowerBombs"
+                      ]}
+                    ]
+                  }}
+                ]}
+              ],
+              "unlock": null,
+              "note": "The clip is done by getting the far right Multiviola in the bottom hallway to climb all the way up. It requires several trips through the hallway, hence killing the Dessgeegas"
+            },
+            {
+              "id": 7,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost":[{"heatFrames": 150}]},
+                {"or":[
+                  {"cost":[
+                    {"enemyDamage": [{
+                      "enemy": "Dessgeega",
+                      "type": "contact",
+                      "hits": 1
+                    }]}
+                  ]},
+                  {"obstacle":{
+                    "id": "F",
+                    "requires":[
+                      {"or":[
+                        {"and":[
+                          "Plasma",
+                          {"cost":[{"heatFrames": 250}]},
+                          {"or":[
+                            "Wave",
+                            {"cost":[{"heatFrames": 200}]}
+                          ]}
+                        ]},
+                        {"and":[
+                          "Morph",
+                          "Power Bomb",
+                          {"cost":[
+                            {"heatFrames": 350},
+                            {"ammo": [
+                              {
+                                "type": "PowerBomb",
+                                "count": 2
+                              }
+                            ]}
+                          ]}
+                        ]},
+                        {"and":[
+                          "Screw Attack",
+                          {"cost":[{"heatFrames": 50}]}
+                        ]}
+                      ]}
+                    ]
+                  }}
+                ]}
+              ],
+              "unlock": null,
+              "note": "Options are to tank the damage or kill the Dessgeegas"
+            }
+          ]
+        },
+        {
+          "from": 7,
+          "to": [
+            {
+              "id": 1,
+              "requires": [
+                {"or":[
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"or":[
+                  {"and":[
+                    {"obstacle":{
+                      "id": "F",
+                      "requires": ["never"]
+                    }},
+                    {"canShineCharge": {
+                      "usedTiles": 22,
```

Comment:
Would you like yet another complication? ;) If you contact damage on the node 6 Dessgeegas there's a ~50% chance you canShineCharge with 31 tiles killing the last 2-4 enemies :P

Reply:
Let's not and say we did?

Reply:
I guess it's relevant as it's a way to get through the room with just PBs and Speed Booster.
I'll make it a direct link from 6 to 1.